### ISSUE TITLE
(Owner) transaction

### ISSUE THREAD

**Issue by AzzaWafiqurrohmah** [AzzaWafiqurrohmah/web-spa#10]
melihat keseluruhan transaksi dari seluruh franchise, ngga kebayan masi tampilannya, tampilannya bakalan ditaruh di sidebar per cabangnya mana aja? atau pakai filter aja?

**Comment by irsyadulibad** [comment]
> melihat keseluruhan transaksi dari seluruh franchise, ngga kebayan masi tampilannya, tampilannya bakalan ditaruh di sidebar per cabangnya mana aja? atau pakai filter aja?

Nah, mending klo ini pke filter aja...klo tiap itu dikasi sidebar ntar kebnyakan
